### PR TITLE
Distance between point and line calculator added

### PR DESCRIPTION
**Pull request by RaveenaBhasin**
## Related Issue #1069 

I have a calculator for finding the distance between point and a line in coordinate geometry calculator section.
Screenshot (1820)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3198,6 +3198,25 @@ <h3>Midpoint </h3><br>
                 class="stopwrap"
                align="right"></p>
             <h4 id="mid_output"> </h4> 
+            <h3>Distance between point and a line</h3><br>
+            <div class="coll">
+                <p>
+                    Line Equation :   <input type="text" id="a" size=3 value='' placeholder="A">  x  +
+                <input type="text" id="b" size=3 value='' placeholder="B"> y + <input type="text" id="c" size=3 value='' placeholder="C"> = 0
+                    <br>
```
Comment (shrey27tri01) [inline index.html:3206]: Can you add another `<br>` tag here so that the "Line Equation" and "Point" paragraphs do not look cluttered?

**Review by shrey27tri01 — COMMENTED** [review]
Please have a look at the inline review. Other than that, everything looks fine.

File: index.html (hunk 1 above)
Comment (RaveenaBhasin) [reply index.html:3206]: Yes I'll add it soon.

**Comment by RaveenaBhasin** [comment]
Updated 
Screenshot (1821)

**Review by sairish2001 — APPROVED** [review]
Will Done :)

**Pull request merged**